sklearn/model_selection/_split.py

```diff
@@ -284,7 +284,16 @@ def __init__(self, n_splits, shuffle, random_state):
 
         self.n_splits = n_splits
         self.shuffle = shuffle
+        # For repr
```

Comment:
I thought about not having this but assumed it would be cleaner to reproduce the input that the user gave in repr instead of random. But your legitimate reasons justify the removal of this... I'll do and push in a while... Thnx